### PR TITLE
Avhw pdr 2

### PR DESCRIPTION
**Pull request by msglazer**
hardware PDR should be ready to add into the site

### CODE REVIEW

File: docs/pdr_avhw.md [hunk 1]
```diff
@@ -0,0 +1,619 @@
+# Flight Hardware
+
+Lead: Matthew Glazer
+
+[TOC]
```
Comment (philiplinden) [inline docs/pdr_avhw.md]: This table of contents is suuuper long. Maybe manually pick out a few links to the top level items and place it here instead.

File: docs/pdr_avhw.md [hunk 2]
```diff
@@ -0,0 +1,619 @@
...
+
+**Overview**: The scope of this PDR covers all hardware design and development as it currently stands for the Nucleus HAB launch. Development approaches, timelines and overall board information are provided. The design of payload, actuation mechanisms and software design are
+discussed in [Altitude Control Logic](pdr_altitudecontrol.md), [Mechanical Systems](placeholder.md) and [Flight Avionics Software](pdr_avsw.md), respectively.
+
+**Overview**: The Flight Hardware system for the nucleus mission provides all the power, computing and control elements required for mission success. There are seven primary PCBs and an assortment of secondary PCBs as part of this system.
+
+### Boards in work:
```
Comment (philiplinden) [inline docs/pdr_avhw.md]: Can probably cut this boards in work section. It was relevant for the review but isn't really relevant on the site and will get stale quick.

File: docs/pdr_avhw.md [hunk 3]
```diff
@@ -0,0 +1,619 @@
...
+This board acts as an interconnect between all payload cards and provides routing for all power and data connections. There is one dedicated card slot for the Main Flight Computer, Communications card and one for the Power Supply Card. In addition there are four payload card slots for various cards. Currently for Nucleus there is an Avionics Sensor Card, Altitude Control/Dynamics Sensor Card for payload cards
+
+Backplane Concepting  - Ethernet Sheet
+
+#### Electrical Interface
+
+##### Connectors
+There is one 40 pin Molex MilliGrid vertical receptacle, one 34 pin Molex MilliGrid vertical receptacle and six 28 pin Molex MilliGrid vertical receptacles providing the interfaces between the various avionics cards. There is also a Molex Microfit header and RJ45 connector for ground station interface and a 10 pin JST header for connection to the balloon board.
+
+###### Main Flight Computer Interface
```
Comment (philiplinden) [inline docs/pdr_avhw.md]: If there's only 1 line of text after a heading, it might work better as a table?

File: docs/pdr_avhw.md [hunk 4]
```diff
@@ -0,0 +1,619 @@
...
+* Dynamics Sensor Nodes
+
+### Communications Card
+
+#### Theory of Operation
+
+###### SDR
+An off the shelf SDR will be mounted to the communications card, it will communicate with the MFC through a USB 2.0 link. 
+
+###### RF Front End
```
Comment (philiplinden) [inline docs/pdr_avhw.md]: comment out empty sections

**Review by philiplinden — CHANGES_REQUESTED** [review]
Consider formatting things as just bold text or putting it in a table rather than headings 5 levels deep.

File: docs/pdr_avhw.md (hunk 2 above)
Comment (msglazer) [reply docs/pdr_avhw.md]: removed

File: docs/pdr_avhw.md (hunk 4 above)
Comment (msglazer) [reply docs/pdr_avhw.md]: done

File: docs/pdr_avhw.md (hunk 1 above)
Comment (msglazer) [reply docs/pdr_avhw.md]: is it better now?

File: docs/pdr_avhw.md (hunk 3 above)
Comment (msglazer) [reply docs/pdr_avhw.md]: i switched it to bold text, with the further sub headings being italics

**Review by msglazer — COMMENTED** [review]
i've realized there's a decent amount of info missing but let's get it live

File: docs/pdr_avhw.md [hunk 5]
```diff
@@ -0,0 +1,672 @@
...
+
+3.7V nominal; 3100mAh cells. Pack would be configured in a 4 series, 2 parallel (4S2P) design with the option of expanding the packs to larger quantities as mission configuration requires. 
+
+Current mission design is a 14.8V nominal (12-16.8V range), 6200mAh pack. Expected temperatures at altitude will reduce total capacity to a currently unknown amount but a conservative approach is an expectation of 40% total capacity loss at max altitude. This can be accounted for using thermal insulation and potentially a small heater.
+
+#### Manufacturing
+
+Battery pack will be tab welded together for electrical connections, cells well be hotglued together for initial mechanical stability with additional framing to be added as necessary.
+
+Testing
```
Comment (philiplinden) [inline docs/pdr_avhw.md]: Should this be a heading or removed?

**Review by philiplinden — APPROVED** [review]

**Comment by msglazer** [comment]
@philiplinden I fixed that testing thing, it was missing its header info

in the battery pack area, changed manufacturing to "pack assembly"

Ill merge this in now

**Pull request merged**